docs/docs/Agents/agent-tool-calling-agent-component.md

```diff
@@ -70,7 +89,11 @@ Add an agent to your problem-solving flow that uses a different OpenAI model for
 The new agent will use `gpt-4o` for the larger tasks of scraping and searching information that requires large context windows.
 The problem-solving agent will now use this agent as a tool, with its unique LLM and toolset.
 
-<img src="/img/tool-calling-agent-as-tool.png" alt="Chat with agent component" style={{display: 'block', margin: 'auto', width: 600}} />
+![The tool calling agent as a tool](/img/tool-calling-agent-as-tool.png)
+
+7. The new agent's metadata can be edited to help the problem-solving agent understand how to use it.
+Click **Edit Tools** to modify the new agent's `name` or `description` metadata so its usage is clear to the problem-solving agent.
+8. For example, the default tool name is `Agent`. Edit the name to `Agent-gpt-4o`, and edit the description to `Use the gpt-4o model for complex problem solving`. The problem-solving agent will understand that this is the `gpt-4o` agent, and will use it for tasks requiring a larger context window.
```

Comment:
Step 8 should not be a numbered step because it's an example for step 7. Make it a paragraph below step 7 instead of a numbered step.